src/builder/create_interaction_response.rs

```diff
@@ -81,11 +81,11 @@ impl CreateInteractionResponseData {
     {
```

Comment:
Rename this method to `create_embed`

Reply:
Should I rename it in `CreateInteractionResponseFollowup` as well? That way the names will match each other.

Reply:
Yes you should